code/datums/gamemode/factions/legacy_cult/ritual.dm

```diff
@@ -0,0 +1,682 @@
+var/runedec = 0 // Rune cap ? 
+
+/obj/effect/rune_legacy
+	desc = "A strange collection of symbols drawn in blood."
+	anchored = 1
+	icon = 'icons/obj/rune.dmi'
+	icon_state = "1"
+	var/visibility = 0
+	layer = RUNE_LAYER
+	plane = ABOVE_TURF_PLANE
+
+	var/dead=0 // For cascade and whatnot.
+
+	var/list/cultwords = list()
+	var/datum/faction/cult/narsie/my_cult
+
+	var/word1
+	var/word2
+	var/word3
+	var/image/blood_image
+
+	var/atom/movable/overlay/c_animation = null
+	var/nullblock = 0
+	var/mob/living/ajourn
+
+	var/summoning = 0
+	var/list/summonturfs = list()
+
+/obj/effect/rune_legacy/New(var/datum/faction/cult/narsie/our_cult)
+	..()
+	blood_image = image(loc = src)
+	blood_image.override = 1
+	for(var/mob/living/silicon/ai/AI in player_list)
+		if(AI.client)
+			AI.client.images += blood_image
+	rune_list_legacy.Add(src)
+	if (!our_cult)
+		our_cult = find_active_faction(LEGACY_CULT)
+	if (!our_cult)
+		CRASH("Creating a rune without an active cult")
+	my_cult = our_cult
+	cultwords = my_cult.cult_words
```

Comment:
`Runtime in ritual.dm, line 42: undefined variable /turf/simulated/floor/var/cult_words`